tests/tests.py

```diff
@@ -6384,9 +6384,19 @@ def test_plantcv_visualize_overlay_two_imgs_size_mismatch():
         _ = pcv.visualize.overlay_two_imgs(img1=img1, img2=img2)
 
 
+@pytest.mark.parametrize("title", ["Include Title", None])
+def test_plantcv_visualize_obj_size_ecdf(title, tmpdir):
```

Comment:
```suggestion
def test_plantcv_visualize_obj_size_ecdf(title):
```

tmpdir isn't needed as there's no file output by the test